lake/commits/object.go

```diff
@@ -21,7 +22,7 @@ type Object struct {
 	Actions []Action    `zed:"actions"`
 }
 
-func NewObject(parent ksuid.KSUID, author, message string, retries int) *Object {
+func NewObject(parent ksuid.KSUID, author, message string, retries int, meta zed.Value) *Object {
```

Comment:
Nit: Put this next to `author` and `message`.
```suggestion
func NewObject(parent ksuid.KSUID, author, message string, meta zed.Value, retries int) *Object {
```

Reply:
makes sense and done! thank you! :)